docs/release-notes.rst

```diff
@@ -6,6 +6,53 @@
  Release Notes
 ###############
 
+**************
+ Version 0.30
+**************
+
+Version 0.30.0
+==============
+
+**Release Date:** April 04, 2024
+
+**Breaking Changes**
+
+-  API: :class:`~determined.pytorch.Trainer` no longer supports the ``Trainer.configure_profiler``
+   option. Profiling is now enabled through the ``Trainer.fit(profiling_enabled=True)`` call.
+
+-  Database migration: system metrics collected by the Determined profiler are now stored in the
+   generic ``metrics`` table. This requires a few schema changes to the ``metrics`` table that will
+   be run during migrations.
```

Comment:
-  Database migration: System metrics collected by the Determined profiler are now stored in the
   generic ``metrics`` table. This requires a few schema changes to the ``metrics`` table that will
   be run during migrations.

Reply:
Others too, don't forget.

Reply:
Capitalized!